contrib/clojure-package/examples/infer/objectdetector/src/infer/objectdetector_example.clj

```diff
@@ -0,0 +1,120 @@
+;; Licensed to the Apache Software Foundation (ASF) under one or more
+;; contributor license agreements.  See the NOTICE file distributed with
+;; this work for additional information regarding copyright ownership.
+;; The ASF licenses this file to You under the Apache License, Version 2.0
+;; (the "License"); you may not use this file except in compliance with
+;; the License.  You may obtain a copy of the License at
+;;
+;;    http://www.apache.org/licenses/LICENSE-2.0
+;;
+;; Unless required by applicable law or agreed to in writing, software
+;; distributed under the License is distributed on an "AS IS" BASIS,
+;; WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+;; See the License for the specific language governing permissions and
+;; limitations under the License.
+;;
+
+(ns infer.objectdetector-example
```

Comment:
This is fantastic to bring to the clojure users 🥇 

Let's think about moving some of this to the unit tests too so we can see if anything breaks in a PR. I think a bit of download is ok and the inference itself doesn't take too long

Reply:
done